.github/workflows/int-test.yml

```diff
@@ -26,11 +26,23 @@ on:
       - 3.0-li-dev**
 
 jobs:
-  clients:
+  test:
     # Name the Job
-    name: clients
+    name: ${{ matrix.module }}
     # Set the type of machine to run on
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false  # We still want to see all jobs' build result to the end
+      matrix:
+        module:
+          - generator
+          - clients
```

Comment:
adding -core here

Reply:
`core` is treated as a separate job, because it has the `prefixes` matrix.

Reply:
Sorry, my mistake.